src/types/form.ts

```diff
@@ -88,6 +88,6 @@ export const schema = z.object({
 
 export type FormType = z.infer<typeof schema>;
 
-const MAX_FILE_SIZE = 500 * 1000; // 5MB
+export const MAX_FILE_SIZE = 500 * 10000; // 5MB
```

Comment:
```suggestion
export const MAX_FILE_SIZE = 5 * 1024 ** 2; // 5MB
```
이렇게 작성해야 하는 것으로 알고 있는데 `500 * 10000`으로 표현한 이유가 궁금해요!